internal/interaction/transaction.go

```diff
@@ -203,7 +229,17 @@ func (s *serviceInteractor) UpdateTransaction(ctx context.Context, tran *entitie
 		requireHistorization = true
 	}
 
-	return s.store.UpdateTransaction(ctx, *tran, requireHistorization)
+	if err := s.store.UpdateTransaction(ctx, *tran, requireHistorization); err != nil {
+		return err
+	}
+
+	// inform the attendee service that a transaction was updated
+	if err := s.attendeeClient.PaymentsChanged(ctx, uint(tran.DebitorID)); err != nil {
```

Comment:
safer to do this only for payments (could theoretically create an infinite loop for dues)